labgrid/driver/sshdriver.py

```diff
@@ -30,8 +30,10 @@ def __attrs_post_init__(self):
         self.logger = logging.getLogger("{}({})".format(self, self.target))
 
     def on_activate(self):
-        self.ssh_prefix = "-i {}".format(os.path.abspath(self.keyfile)
+        self.ssh_prefix = " -i {}".format(os.path.abspath(self.keyfile)
```

Comment:
Is this change with the change below necessary? Looks like an unnecessary change to me.